content/en/docs/tasks/policy-enforcement/rate-limit/index.md

```diff
@@ -0,0 +1,285 @@
+    spec:
+      workloadSelector:
+        # select by label in the same namespace
+        labels:
+          istio: ingressgateway
+      configPatches:
+        # The Envoy config you want to modify
+        - applyTo: HTTP_FILTER
+          match:
+            context: GATEWAY
+            listener:
+              filterChain:
+                filter:
+                  name: "envoy.http_connection_manager"
+                  subFilter:
+                    name: "envoy.router"
+          patch:
+            operation: INSERT_BEFORE
+            # Adds the Envoy Rate Limit Filter in HTTP filter chain.
+            value:
+              name: envoy.ratelimit
+              typed_config:
+                "@type": type.googleapis.com/envoy.config.filter.http.rate_limit.v2.RateLimit
+                # domain can be anything! Match it to the ratelimter service config
+                domain: productpage-ratelimit
+                failure_mode_deny: true
+                rate_limit_service:
+                  grpc_service:
+                    envoy_grpc:
+                      cluster_name: rate_limit_cluster
+                    timeout: 10s
+        - applyTo: CLUSTER
+          match:
+            cluster:
+              service: ratelimit.default.svc.cluster.local
+          patch:
+            operation: ADD
+            # Adds the rate limit service cluster for rate limit service defined in step 1.
+            value:
+              name: rate_limit_cluster
+              type: STRICT_DNS
+              connect_timeout: 10s
+              lb_policy: ROUND_ROBIN
+              http2_protocol_options: {}
+              load_assignment:
+                cluster_name: rate_limit_cluster
+                endpoints:
+                - lb_endpoints:
+                  - endpoint:
+                      address:
+                        socket_address:
+                          address: ratelimit.default.svc.cluster.local
+                          port_value: 8081
+    ---
+    apiVersion: networking.istio.io/v1alpha3
+    kind: EnvoyFilter
+    metadata:
+      name: filter-ratelimit-svc
+      namespace: istio-system
+    spec:
+      workloadSelector:
+        labels:
+          istio: ingressgateway
+      configPatches:
+        - applyTo: VIRTUAL_HOST
+          match:
+            context: GATEWAY
+            routeConfiguration:
+              vhost:
+                name: "*:80"
+                  route:
+                    action: ANY
+          patch:
+            operation: MERGE
+            # Applies the rate limit rules.
+            value:
+              rate_limits:
+                - actions: # any actions in here
+                  - request_headers:
+                      header_name: ":path"
+                      descriptor_key: "PATH"
+    EOF
+    {{< /text >}}
+
+### Local Rate Limit
+
+Envoy supports using Local Rate Limit on connection and HTTP level. This helps applying rate limit at the instance level
```

Comment:
I think we probably need to emphasize with more text that this limit is enforced by the proxy itself, with no communication with other services or proxies. This is implied, but I think it will help to be fully explicit here (maybe with examples) ?

Reply:
Done.